exporters/committime/collector_base.py

```diff
@@ -191,7 +225,8 @@ def get_metric_from_build(self, build, app, namespace, repo_url):
 
             metric.commit_hash = commit_sha
             metric.name = app
-            metric.committer = build.spec.revision.git.author.name
+            # metric.committer = build.spec.revision.git.author.name
+            metric.committer = "default"
```

Comment:
why not label for this one?

Reply:
@etsauer for some reason, in my `build`  object the property `build.spec.revision.git.author.name` doesn't exist... so that the committime exporter does not fail, I set it to "default" value.

Reply:
It might only be there if it was kicked off from a `BuildConfig`. I'll make it try but then fall back.